### ISSUE TITLE
Roadmap for our approaches to code

### ISSUE THREAD

**Issue by Bjwebb** [OpenDataServices/developer-docs#27]
What things we're currently trailing - e.g. Docker, Agile approaches
What's on our wishlist e.g. Email signing

https://github.com/OpenDataServices/developer-docs/issues/21 - general approaches to software development may be relevant to this.

**Comment by caprenter** [comment]
I don't think we are currently trialling very much - or maybe we are. Flagging this for review
